css/style.css

```diff
@@ -303,61 +326,116 @@ hr {
   font-family: "Roboto", serif;
   font-style: normal;
   font-weight: 350px;
-  font-size: 32px;
+  font-size: 24px;
   line-height: 44px;
   text-align: left;
   color: #fff;
   padding: 10px 0;
-  margin: 20px 25px;
-  margin-right: 24px;
+  margin: 20px 10px;
+  margin-left: 800px;
+  margin-top: -320px;
+}
+
+.title-post-left h2 {
+  font-family: 'Roboto', serif;
+  font-style: normal;
+  font-weight: 350;
+  font-size: 24px;
```

Comment:
Hello @maherramadan78 👋, I've noticed that you are using the correct font-family: "Roboto", but I didn't find any import of it on your project, so the default one is being applied instead of it.
To use custom fonts, you will need to first add them to your project. The easiest way to do it is by adding the font to your HTML `head`. I will explain to you a way of doing this using [Google Fonts](https://fonts.google.com/) 😄: 

- Search for the font that you want: 
![image](https://user-images.githubusercontent.com/42799579/198803592-001b8e4a-8f09-40bb-b094-d8196a4bcdab.png) 

- Select it: 
![image](https://user-images.githubusercontent.com/42799579/198803724-bb8ef580-55e5-454c-a351-25f326eb2992.png) 

- You will be taken to the font page, scroll down until you have the weights: 
![image](https://user-images.githubusercontent.com/42799579/198804057-63e483f4-fc4e-469b-b9ac-cf6d611be885.png) 

- Select the ones that you want by clicking here: 
![image](https://user-images.githubusercontent.com/42799579/198805635-c8e31d9e-82f6-4c72-8bdf-ffe170f5683f.png) 

- A box will appear on the right, after selecting all the fonts that you want, copy the link: 
![image](https://user-images.githubusercontent.com/42799579/198804518-6865e9ea-f5da-4636-9778-a7b3f6bc12a2.png) 

And add the copied link to your HTML `head` 😉. 
> Take into account that you can select multiple fonts & weights 😃

Reply:
ok , its done . thank you